src/jaegertracing/UDPTransport.cpp

```diff
@@ -27,6 +27,10 @@
 #include <thrift/protocol/TCompactProtocol.h>
 #include <thrift/transport/TBufferTransports.h>
 
+#ifdef _MSC_VER
+#pragma warning(disable : 4267)
```

Comment:
it would be useful to add comments about why something is being disabled

Reply:
ok